spec.html

```diff
@@ -4259,7 +4259,7 @@
         1. Let _keys_ be ? _O_.[[OwnPropertyKeys]]().
         1. If _level_ is `"sealed"`, then
           1. Repeat for each element _k_ of _keys_,
-            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{ [[Configurable]]: *false*}).
+            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{[[Configurable]]: *false*}).
```

Comment:
Right, I'm not just talking about `PropertyDescriptor`, i'm talking about every curly braced construct that's mimicking an object literal.

Personally, I think a space both before and after the left brace (and a space inside the right brace) is the most readable.